src/command.rs

```diff
@@ -1,7 +1,7 @@
+use serde_json::{self, json};
 use std::env;
 use std::io;
 use std::process;
```

Comment:
Let's keep the order of imports as is, i.e:
1. `std`
2. External crates
3. Local crates

```suggestion
use std::env;
use std::io;
use std::process;
use serde_json::{self, json};
```